lib/resolver.ex

```diff
@@ -0,0 +1,49 @@
+defmodule Gannbaruzoi.Resolver do
```

Comment:
I think we can separate the files here.
- lib/resolver/task
- lib/resolver/log